core/store/migrate/migrations/0215_functions_subscriptions.sql

```diff
@@ -0,0 +1,17 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE functions_subscriptions(
```

Comment:
We need to namespace this. How about adding router contract address? We could also add ToS contract address but I'm not sure if that's useful ...